openqabot/types/incidents.py

```diff
@@ -96,35 +96,37 @@ def __call__(
                             inc.id,
                         )
                         continue
+
+                    if inc.staging:
+                        continue
+
+                    flavor_settings = self.settings.copy()
```

Comment:
We can potentially further optimize the code in regard of this copy and the one in 
https://github.com/openSUSE/qem-bot/pull/145/files#diff-81a88cee51bd2524a7e938a6b6d53d79f526ffb436071ead2d3d83163b38b1ecR255